### PR TITLE
bump purescript version to 0.13.3

### PR DESCRIPTION
**Pull request by colinwahl**
The network library version was bumped for `purescript` here: https://github.com/purescript/purescript/pull/3597

The changes to the codegen function are from: https://github.com/purescript/purescript/pull/3645

Also, thanks for making this tool!

### CODE REVIEW

**Comment by colinwahl** [comment]
I'm going to look into the hlint warnings later today

File: zephyr.cabal [hunk 1]
```diff
@@ -57,7 +57,7 @@ library
   build-depends:      
       aeson >=1.0 && <1.4
     , ansi-terminal >=0.7.1 && <0.9
-    , base >=4.8 && < 4.12
+    , base >=4.8
```
Comment (coot) [inline zephyr.cabal]: It's probably better to leave '&& < 4.13` bound.

File: src/Language/PureScript/DCE/Foreign.hs [hunk 2]
```diff
@@ -103,11 +103,23 @@ dceForeignModule is stmts = filter filterExports stmts
...
   isUsedInStmt n (JSStatementBlock _ ss _ _) = any (isUsedInStmt n) ss
+  isUsedInStmt n (JSLet _ es _) = isUsedInExprs n es
   isUsedInStmt n (JSDoWhile _ stm _ _ e _ _) = isUsedInStmt n stm || isUsedInExpr n e
   isUsedInStmt n (JSFor _ _ es1 _ es2 _ es3 _ s) = isUsedInExprs n es1 || isUsedInExprs n es2 || isUsedInExprs n es3 || isUsedInStmt n s
   isUsedInStmt n (JSForIn _ _ e1 _ e2 _ s) = isUsedInExpr n e1 || isUsedInExpr n e2 || isUsedInStmt n s
   isUsedInStmt n (JSForVar _ _ _ es1 _ es2 _ es3 _ s) = isUsedInExprs n es1 || isUsedInExprs n es2 || isUsedInExprs n es3 || isUsedInStmt n s
   isUsedInStmt n (JSForVarIn _ _ _ e1 _ e2 _ s) = isUsedInExpr n e1 || isUsedInExpr n e2 || isUsedInStmt n s
+  isUsedInStmt n (JSForLet _ _ _ es1 _ es2 _ es3 _ s) =
+    isUsedInExprs n es1 || isUsedInExprs n es2 || isUsedInExprs n es3 ||
+    isUsedInStmt n s
```
Comment (coot) [inline src/Language/PureScript/DCE/Foreign.hs]: If it fits in 80 chars, could you put this on the previous line?

**Comment by colinwahl** [comment]
@coot I've addressed your feedback. Let me know if there's anything else I need to do to get this merged!

**Comment by thomashoneyman** [comment]
Tested this out on our work codebase and looks to be working just fine. 👍

**Pull request merged**